tests/unit/session.test.ts

```diff
@@ -49,72 +99,129 @@ describe('Session Service Unit Tests', () => {
     );
   });
 
-  test('should throw ValidationError if issuer is mismatched', async () => {
-    (jwtVerify as jest.Mock).mockResolvedValue({
-      payload: {
-        iss: 'https://invalid-issuer.com',
-        sub: TEST_USER_ID,
-        name: TEST_FULL_NAME,
-      },
+  test('Invalid Issuer 1', async () => {
```

Comment:
we must decide for naming here, either you write some "status" like "Invalid Issuer 1" or you use this "should xxx". whats the recommendation for the unit test library?